redhat-cop/businessautomation-cop
# Restoring pam-eap-setup handling of RHPAM.7.12

#### What is this PR About?
pam-eap-setup can again install RHPAM.7.12 along with earlier versions of RHPAM

#### How do we test this?
pam-eap-setup should be able to install all GA versions of RHPAM

cc: @redhat-cop/businessautomation-cop
